src/sentry/api/endpoints/admin_project_configs.py

```diff
@@ -0,0 +1,43 @@
+from django.http import Http404
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, pending_silo_endpoint
+from sentry.api.permissions import SuperuserPermission
+from sentry.models import Project
+from sentry.relay import projectconfig_cache
+
+
+@pending_silo_endpoint
+class AdminRelayProjectConfigsEndpoint(Endpoint):
+    private = True
+    permission_classes = (SuperuserPermission,)
+
+    def get(self, request: Request) -> Response:
+        project_id = request.GET.get("projectId")
+
+        project_keys = []
+        if project_id is not None:
+            try:
+                project = Project.objects.get_from_cache(id=project_id)
+                for project_key in project.key_set.all():
+                    project_keys.append(project_key.public_key)
+
+            except Exception:
+                raise Http404
+
+        project_key = request.GET.get("projectKey")
+        if project_key is not None:
+            project_keys.append(project_key)
+
+        configs = {}
+        for key in project_keys:
+            cached_config = projectconfig_cache.get(key)
+            if cached_config is not None:
+                configs[key] = cached_config
+            else:
+                configs[key] = "EMPTY"
```

Comment:
```suggestion
                configs[key] = None
```

Would using `None`/`null` be a better sentinel value to indicate that the project config is not cached?

Reply:
I debated a while about this, since the API is to be used in a rather primitive UI and the string will be dumped in a simple viewer I thought that when there are many configs I would like a quick way to search for missing configs, and null might appear in many places... I'll leave it as is for now, but we may change it if we can sort it in a better way on the UI side.

Reply:
On a second thought I'll make it None :-)